.github/workflows/general.yml

```
@@ -29,7 +29,7 @@ jobs:
         with:
           components: clippy
       - name: Linting
-        run: cargo clippy -- -D warnings
+        run: cargo clippy --all-targets -- -D warnings
```

We can also run it for all the features:
```suggestion
        run: cargo clippy --all-targets --all-features -- -D warnings
```